rockPaperScissors.py

```diff
@@ -3,48 +3,51 @@
 
 import random
 
-def gameLogic(player, computer, playerScore, computerScore):
-    playerIndex = RPS.index(player)
-    computerIndex = RPS.index(computer)
```

Comment:
RPS was defined outside of this function scope, which made this function impure. If someone changed the code so that RPS had different elements in it, this function would break even though you didn't change any code in the function. trust me, it sucks when that happens.